README.md

```diff
@@ -41,9 +41,9 @@ We are willing to answer questions.
 ## Getting Started
 
 ```javascript
-import vjsn from "zome-vjsn";
+import vueSpatialNavigation from "zome-vjsn";
 
-app.use(vjsn, globalConfig);
```

Comment:
`globalConfig` was not declared in the previouses lines (undefined error), so I removed it, as it was optional.